pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -397,11 +397,26 @@ func (dm *DockerManager) inspectContainer(id string, podName, podNamespace strin
 		parseTimestampError("FinishedAt", iResult.State.FinishedAt)
 	}
 
+	// default to the image ID, but try and inspect for the RepoDigests
+	imageID := DockerPrefix + iResult.Image
+	imgInspectResult, err := dm.client.InspectImage(iResult.Image)
+	if err != nil {
```

Comment:
hmm... like

```go
switch {
case err != nil:
  utilruntime.HandleError(fmt.Errorf("unable to inspect docker image %q while inspecting docker container %q: %v", containerName, iResult.Image, err))
case len(imgInspectResult.RepoDigests) > 1:
  glog.V(4).Infof("Container %q had more than one associated RepoDigest (%v), only using the first", containerName, imgInspectResult.RepoDigests)
  fallthrough
case len(imgInspectResult.RepoDigests) > 0:
  imageID = xyz
```
Maybe?  I could also just move the logging if statement out a bit (it doesn't need to be nested like it is).  I'm not convinced that a switch is better than that